lib/handlers/PersonCreatorHandler.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../models/Teacher'
+require_relative '../models/Student'
+
+class PersonCreator
+  def add_student(age:, name:, parent_permission: true)
+    Student.new(age: age, name: name, parent_permission: parent_permission)
+  end
```

Comment:
- [ ] As sighted by the previous reviewer, i tried adding with an age of `17` and below, and they both resulted in an error when picking yes or no in the options. Kindly review the error below, which has to so with the conditional application your specified for that option 👍 

### Error view 👍 
![image](https://github.com/PabloEGonz/school-library/assets/50990925/db3c0882-8617-44bb-b3a2-f39a8927958f)